### ISSUE TITLE
Cult 3 armor with pale skin is silly shit

### ISSUE THREAD

**Issue by aceedex** [vgstation-coders/vgstation13#22102]
pale skin makes you stun resist right? so the only way to stop someone is with damage
cult armor makes it that you can survive 10 or so hits from a mecha laser cannon
and as always you can make diona/mushroom/golem cult

Its manifested ghost 3.0 combined and I dont like it

**Comment by Irradiation** [comment]
(Helmets have the same stats more or less as armor)
>/obj/item/clothing/suit/space/cult
	name = "cult armor"
	desc = "A bulky suit of armor bristling with spikes. It looks space proof."
	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/cultstuff.dmi', "right_hand" = 'icons/mob/in-hand/right/cultstuff.dmi')
	icon_state = "cultarmor"
	item_state = "cultarmor"
	w_class = W_CLASS_MEDIUM
	allowed = list(/obj/item/weapon/tome,/obj/item/weapon/melee/cultblade,/obj/item/weapon/melee/soulblade,/obj/item/weapon/tank,/obj/item/weapon/tome,/obj/item/weapon/talisman,/obj/item/weapon/blood_tesseract)
	slowdown = HARDSUIT_SLOWDOWN_MED
	armor = list(melee = 60, bullet = 50, laser = 50,energy = 15, bomb = 50, bio = 30, rad = 30)
	siemens_coefficient = 0
	species_fit = list(VOX_SHAPED, UNDEAD_SHAPED)
	clothing_flags = PLASMAGUARD|CONTAINPLASMAMAN|ONESIZEFITSALL
	max_heat_protection_temperature = FIRESUIT_MAX_HEAT_PROTECTION_TEMPERATURE

**Comment by iFlashYou** [comment]
I ded, pls nerf

**Comment by Hoshi-chan** [comment]
Literally based on observing a mecha killing a roomful of cultists and constructs, wasting time firing clusterbangs against pale cultists, and failing to kill _all_ cultists, only some.

**Comment by aceedex** [comment]
Dont project your bad habits on me, marko. I was a ghost from the start. Its also not the only round where Ive observed as such. There was another with a sec officer, 5 crew members and a borg pew pewing a cultist with pale skin and that cultist simply killed them all except for one who just said fuck it and left

Its also funny how you cant see pale skin through cult armor. you know, the supposed big downside of pale skin

Again, its not JUST the armor I take issue. I take issue with the entire combo.

**Comment by Allspice** [comment]
> There was another with a sec officer, 5 crew members and a borg pew pewing a cultist with pale skin and that cultist simply killed them all except for one who just said fuck it and left

Dunno if it's the same round, but the last time I heard about that they were firing on him with tasers and he just used a sword to whack em.

**Comment by DeityLink** [comment]
>cult armor makes it that you can survive 10 or so hits from a mecha laser cannon

Fun fact, security officer armor and helmet have better resistance to lasers than cult armor/helmet 
![image](https://user-images.githubusercontent.com/7573912/54721344-08eae200-4b62-11e9-861b-33fcf5bd458a.png)

sj_benster was probably near crit, I'll try and reproduce that in the thunderdome.

**Comment by DeityLink** [comment]
By the way comparing this to old manifested ghost is a travesty.

**Comment by DeityLink** [comment]
Testing in the thunderdome reports that a manifested ghost wearing cult armor will still die most of the time before even reaching 10 heavylaser hits. manifested ghosts wearing security officer armor can take 1-2 more hits as well on average.

But the entire nature of this test has to be taken with the consideration that armor code is stupid and a hit will either randomly be ignored, halved, or taken without reduction.

TL;DR: sj_benster didn't have just pale body and cult armor, but a load of luck on his side.
